Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -127,10 +127,49 @@ namespace PlayerCommands
 		pub::Player::PopUpDialog(client, caption, message, POPUPDIALOG_BUTTONS_CENTER_OK);
 	}
 
+	bool PreventTooOftenLogins(uint client, PlayerBase *base, wstring charname)

```

Comment:
How about RateLimitLogins? "Prevent too often logins" is not great grammar

Reply:
Fixed